test/smoke_test/test-array-changed-in-callee.c

```diff
@@ -1,5 +1,5 @@
 // RUN: clang -o %t-opt %s
-// RUN: llvm-mctoll -d --include-files="/usr/include/stdio.h,/usr/include/stdlib.h" %t-opt
+// RUN: llvm-mctoll -d -I /usr/include/stdio.h -I /usr/include/stdlib.h %t-opt
```

Comment:
I had added `-I` as an alias for `--include-files=<single-header-file>` and works in the current `master` branch. I understand  your change  continues  to support it.

But, I'd like to keep the option `--include-files=<comma-separated-list-of-header-files>`. So  changing the test is not preferable.

Is there no way to support such options in `Opts.td`?

Reply:
Flag added